package.json

```diff
@@ -2,13 +2,17 @@
   "name": "@counterfactual/truffle-box",
   "version": "0.0.0",
   "description": "",
-  "main": "src/app.js",
+  "main": "truffle.js",
```

Comment:
why was this changed?

Reply:
To match how it's done [here](https://github.com/truffle-box/pet-shop-box/blob/master/package.json#L5).